assets/css/sensei-course-theme/lesson-complete-transition.scss

```diff
@@ -0,0 +1,35 @@
+@keyframes sensei-course-theme-lesson-completion-notice-fadein {
+	from { opacity: 0; }
+	to { opacity: 1; }
+}
+
+.sensei-course-theme-lesson-completion-notice {
+	position: fixed;
+	z-index: 1;
+	height: calc(100vh - var(--full-header-height));
+	left: var(--sidebar-width);
+	right: 0;
+	display: flex;
```

Comment:
Contact teacher has a similar message, but I noticed that was done with specific classes to use in other contexts out of the Course Theme too, so I did something specific here too. If we create more notices like that in the future, we can refactor it to centralize in someplace.